index.bs

```diff
@@ -1268,6 +1268,9 @@ interface WebTransportSendStream : WritableStream {
 A {{WebTransportSendStream}} is always created by the
 [=WebTransportSendStream/create=] procedure.
 
+The {{WebTransportSendStream}}'s [=transfer steps=] and
+[=transfer-receiving steps=] are those of {{WritableStream}}.
```

Comment:
If you can get Bikeshed to play along, it would be helpful to link to https://streams.spec.whatwg.org/#ws-transfer.